### PR TITLE
Connecting Dashboard to contentful, dynamic variables

### PR DESCRIPTION
**Pull request by deadlybutter**
This PR grabs content from Contentful to fill the dashboard.

screen shot 2017-04-12 at 10 50 06 am

Additionally, it adds some dynamic variables which let the campaign lead replace text with stuff like the **end date** or the **total signups**. This is what that looks like in Contentful

screen shot 2017-04-12 at 10 51 21 am

### CODE REVIEW

**Comment by weerd** [comment]
@deadlybutter I saw in Contentful that the end date is required. Are we sure that we want it required? Are there any "evergreen" campaigns we'll have on here? Not actually sure...

File: resources/assets/components/Dashboard/index.js [hunk 1]
```diff
@@ -2,26 +2,40 @@ import React from 'react';
...
+
   return (
     <Flex>
       <FlexCell width='full'>
         <div className='dashboard'>
           <div className='dashboard__block -quarter'>
-            <h1>45 days</h1>
-            <span>until campaign closes</span>
+            <h1>{ replaceTemplateVars(props.content.fields.leftValue) }</h1>
+            <span>{ replaceTemplateVars(props.content.fields.leftDescription) }</span>
```
Comment (weerd) [inline resources/assets/components/Dashboard/index.js]: Not super keen on these fields being named based on visual location...

**Comment by deadlybutter** [comment]
@weerd I can un-require it, good point!

File: resources/assets/components/Dashboard/index.js (hunk 1 above)
Comment (deadlybutter) [inline resources/assets/components/Dashboard/index.js]: Any suggestions? Was drawing a blank on this one

Comment (weerd) [inline resources/assets/components/Dashboard/index.js]: So one of them typically holds "timeline" value and the other holds "member count" related data? Ideally something generic. Any suggestions @DFurnes. I could just see a scenario where we A/B test switching order around on dashboard stuff and that'll all just get confusing if one is name `Left Value` but in future shows up in the center or something (if) as the dashboard changes in design.

Comment (DFurnes) [inline resources/assets/components/Dashboard/index.js]: I think I'd be more in favor of `first` and `second` than left/right, but I can't really think of anything better than that. The problem is we don't really want to lock ourselves in with the semantic content here either, so I'm not sure there's a great name to be found. 😕

Comment (weerd) [inline resources/assets/components/Dashboard/index.js]: "Timeline" & "MemberTally"

Comment (weerd) [inline resources/assets/components/Dashboard/index.js]: @DFurnes hmmm, true. 🤔  Yeah, maybe `first` and `second` could work better...

**Comment by deadlybutter** [comment]
@weerd @DFurnes Renamed!

**Pull request merged**